src/commodore/index.ts

```diff
@@ -87,20 +98,49 @@ export async function extractPackageFile(
   let components: CommodoreComponentDependency[];
 
   const repoDir: string | undefined = getGlobalConfig().localDir;
+  let globalDir: string = '';
+  // Tenant repos must have `commodore.tenantId` set
+  const isTenantRepo: boolean =
+    config.tenantId !== undefined && config.tenantId != '';
+
+  let cluster: ClusterData = new ClusterData();
+  let globalExtraConfig: any = {};
+
+  if (isTenantRepo) {
+    logger.debug('Identified current repo as tenant repo');
+    cluster.name = parse(fileName).name;
+    cluster.tenant = config.tenantId;
+
+    logger.info(
+      {
+        fileName: fileName,
+        globalRepo: config.globalRepoURL,
+        tenantId: config.tenantId,
+      },
+      `Global repo for this tenant is not initialized yet, cloning it`
+    );
```

Comment:
This log line is a bit misleading. We will print this all the time, even if the repo is initialized.

Reply:
moved it into `cloneGlobalRepo`